webapp/src/utils/config.spec.ts

```diff
@@ -1,149 +1,328 @@
-import { LyraConfigReadingError } from '@/errors';
 import mock from 'mock-fs';
 import { describe, expect, it } from '@jest/globals';
-import LyraConfig, { MessageKind } from './config';
+import { LyraConfig, MessageKind, ServerConfig } from './config';
+import {
+  LyraConfigReadingError,
+  ProjectNameNotFoundError,
+  ProjectPathNotFoundError,
+  ServerConfigReadingError,
+} from '@/errors';
 
-describe('LyraConfig', () => {
-  describe('readFromDir()', () => {
-    it('reads message kind and path from lyra.yml', async () => {
-      mock({
-        '/path/to/repo/lyra.yml': [
-          'projects:',
-          '- path: .',
-          '  messages:',
-          '    format: yaml',
-          '    path: locale',
-          '  translations:',
-          '    path: locale',
-        ].join('\n'),
+describe('config.ts', () => {
+  describe('LyraConfig', () => {
```

Comment:
Same thing here. I don't like it when changes like these are made easily. Indenting an entire file is not a minor thing, and I'm honestly not sure I understand the need to have a `describe('config.ts')`. What information does that provide about the code, other than the file the tested code is in (which is already revealed by the name of the test file).

If you want to test two classes in a single test file, you can just have two top-level `describe()`, right?

But ultimately, I think `ServerConfig` should be in a separate file.

Reply:
usually for me the point of having one describe that wrap all file and named after the name are:
1. For some IDE like WebStorm it makes it easy by one click to run all unit test in this file
2. AFAIK there is no command in Jest to run a test file but it only can be called by it describe

in all cases this will be split into 2 files as you mentioned above to follow splitting of config.ts

Reply:
done seperate